.ci/cico_chectl_minikube.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+set -e -x
+
+#Stop execution on any error
+trap "fail_trap" EXIT
+
+init() {
+  SCRIPT=$(readlink -f "$0")
+  SCRIPTPATH=$(dirname "$SCRIPT")
```

Comment:
You are using undescore everywhere, so maybe SCRIPTPATH => SCRIPT_PATH . And name of this variable for me associated with full path to the script, maybe name SCRIPT_DIR will be better?

Reply:
ok, I'll change.

Reply:
Sound correct for me.